packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -681,6 +682,14 @@ export const buildCss = (node: t.Expression | t.Expression[], meta: Metadata): C
     return buildCss(node.quasi, meta);
   }
 
+  if (
+    isCompiledCSSObjectLiteral(node, meta) &&
+    node.arguments.length === 1 &&
+    t.isObjectExpression(node.arguments[0])
+  ) {
+    return buildCss(node.arguments[0], meta);
```

Comment:
/nit You could just pass in `node.arguments` if you cast it, since the types should pick up incorrect `css` usage. Either way is fine though.